.github/workflows/release.yml

```diff
@@ -35,4 +35,8 @@ jobs:
         run: pip3 install ansible-base
 
       - name: Trigger a new import on Galaxy.
-        run: ansible-galaxy role import --api-key ${{ secrets.GALAXY_API_KEY }} $(echo ${{ github.repository }} | cut -d/ -f1) $(echo ${{ github.repository }} | cut -d/ -f2)
+        run: >-
```

Comment:
Can you revert this change, only because it's unrelated and is standardized across all 60+ of my Galaxy roles.

Reply:
Done. That was the linter though...